yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/sync/SyncTransactionalClientImpl.java

```diff
@@ -115,6 +116,11 @@ public <T> List<T> lookupRows(
         return client.lookupRows(req, serializer).join();
     }
 
+    @Override
+    public List<UnversionedRowset> multiLookup(MultiLookupRequest req) {
```

Comment:
As I mentioned earlier, I renamed method back to multiLookup (probably, you got mistaken in your comment because of my various ways to name this method)